clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/DockerIntegrationSpec.scala

```diff
@@ -100,6 +100,10 @@ class CoreDockerUbamSpec extends DockerIntegrationSpec("Clio in Docker") with Ub
 class CoreDockerCramSpec extends DockerIntegrationSpec("Clio in Docker") with WgsCramTests
 class CoreDockerGvcfSpec extends DockerIntegrationSpec("Clio in Docker") with GvcfTests
 
+class CoreDockerArraysSpec
+    extends DockerIntegrationSpec("Clio in Docker")
+    with ArraysTests
+
```

Comment:
Why is this formatted differently from the lines above it?

Reply:
only scalafmt knows

Reply:
sigh